tools/teradata-bigquery-sql-translator/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
The LICENSE file isn't needed here as the subdirectory will be covered by the root license.